homeassistant/components/tado/services.yaml

```diff
@@ -24,6 +16,26 @@ set_climate_timer:
           max: 100
           step: 0.5
           unit_of_measurement: '°'
+    time_period:
+      name: Time period
+      description: Set the time period for the boost. NB dont set this and the overlay
+      required: false
+      example: "01:30:00"
+      default: "01:00:00"
+      selector:
+        text:
+    requested_overlay:
```

Comment:
It's weird that a service called `set_climate_timer` allows setting a `requested_overlay` which is not compatible with setting a time period.

Reply:
Yeah I get what you mean, but the reality is that set_climate_timer is actually setting an overlay in tado, all I am adding is the ability to choose which type of overlay. Probably the service should of been called set_overlay, but I dont want to change that due to backwards compatibility